src/inc/msquic.h

```diff
@@ -1714,6 +1725,9 @@ typedef struct QUIC_API_TABLE {
     QUIC_CONNECTION_COMP_RESUMPTION_FN  ConnectionResumptionTicketValidationComplete; // Available from v2.2
     QUIC_CONNECTION_COMP_CERT_FN        ConnectionCertificateValidationComplete;      // Available from v2.2
 
+    QUIC_CONNECTION_OPEN_IN_PARTITION_FN
```

Comment:
Should we make this preview, even if only for a short time?

Reply:
As discussed over IM, I've decided that I don't want to.